posts/2024-09-05-Rust-1.81.0.md

```diff
@@ -45,7 +45,7 @@ requirements documented in [PartialOrd] and [Ord].
 ### `#[expect(lint)]`
 
 1.81 stabilizes a new lint level, `expect`, which allows explicitly noting that
-a particular lint *should* occur, and warning if it doesn't.  The intended use
+a particular lint *should* occur, and warn if it doesn't.  The intended use
```

Comment:
If we're going to not use the gerund here then it needs to agree with the singular "lint level".

```suggestion
a particular lint *should* occur, and warns if it doesn't.  The intended use
```

i.e. "lint level, `expect`, which [...] warns if it doesn't"

Though I don't agree that this needs changing at all. Or at least this should be split into two separate sentences if it's ambiguous.

Reply:
The "warn" is not marked because there's an auxiliary, "should".